src/main.rb

```diff
@@ -0,0 +1,154 @@
+require_relative 'app'
+
+def greeting
+  commands = [
+    '[1] List all books',
+    '[2] List all people',
+    '[3] Create person',
+    '[4] Create book',
+    '[5] Create rental',
+    '[6] List rentals by person',
+    '[7] Quit'
+  ]
+  commands.each { |cmd| puts cmd }
+  puts 'Make your choice to continue...'
+end
+
+def user_choice(choice, app) # rubocop:disable Metrics/CyclomaticComplexity
+  case choice
+  when '1'
+    app.list_all_books
+  when '2'
+    app.list_all_people
+  when '3'
+    create_person(app)
+  when '4'
+    create_book(app)
+  when '5'
+    create_rental(app)
+  when '6'
+    list_rentals_by_id(app)
+  when '7'
+    return true
+  else
+    'Believe in your self! Press the right key!'
+  end
+  false
+end
+
+def create_person(app)
+  puts "To create a [Student] press [1]\nTo create a [Teacher] press [2]"
+  choice = gets.chomp
+  case choice
+  when '1'
+    create_student(app)
+  when '2'
+    create_teacher(app)
+  else
+    'Select person category first'
+  end
+end
+
+def create_student(app)
+  puts 'Enter Student [Name]'
+  name = gets.chomp
+  puts 'Enter Student [Age]'
+  age = gets.chomp
+  app.create_student(age, name)
+  puts "New Student added:\n[Student] ID: #{app.students.last.id}, "
+  print "Name: #{app.students.last.name}, Age: #{app.students.last.age}"
+end
```

Comment:
- [ ] for a student the app is supposed to check if they have parent permission.

Reply:
# Improvements
- check if age < 18
- require setting permission
- if age >= 18 sets permission to true automatically

Thank you for your advice and assistance.

Reply:
Well resolved 👍🏽